frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -430,6 +430,54 @@ describe("scenarios > question > nested", () => {
       });
     });
   });
+
+  it.skip("'distribution' should work on a joined table from a saved question (metabase#14787)", () => {
+    // Set the display really wide and really tall to avoid any scrolling
+    cy.viewport(1600, 1200);
+
+    ordersJoinProducts("14787");
+    // This repro depends on these exact steps - it has to be opened from the saved questions
+    cy.visit("/question/new");
+    cy.findByText("Simple question").click();
+    cy.findByText("Saved Questions").click();
+    cy.findByText("14787").click();
+
+    // The column title
+    cy.findByText("Products → Category").click();
+    cy.findByText("Distribution").click();
+    cy.contains("Summarize").click();
+    cy.findByText("Group by")
+      .parent()
+      .within(() => {
+        cy.log("**Regression that worked on 0.37.9**");
+        isSelected("Products → Category");
+      });
+
+    // Although the test will fail on the previous step, we're including additional safeguards against regressions once the issue is fixed
+    // It can potentially fail at two more places. See [1] and [2]
+    cy.get(".Icon-notebook").click();
+    cy.get("[class*=NotebookCellItem]")
+      .contains("Products → Category") /* [1] */
```

Comment:
Wouldn't that succeed no matter what, since it's `contains()` and the problematic value is
`Products → Products → Category` ?
It's not super important, since there's the two `isSelected()` checks.

Reply:
Good catch.
This would probably give a false positive, you're right.

Reply:
Fixed it :)

Reply:
That doesn't fix it. It's still `contains()` - it should have been a negative assertion - whatever Cypress lingo is for this:
```
.notContains("Products → Products → Category") /* [1] */ 
```

Reply:
Sanity check - this yielded only 1 element (although at least 3 elements `contain` ID) so it seems it works:
![image](https://user-images.githubusercontent.com/31325167/107831042-7541e080-6d8d-11eb-808e-95231b4a537d.png)

Reply:
But you are trying to make sure that `Products → Products → Category` does **not** exist, right?
https://user-images.githubusercontent.com/1447303/107767184-e43f1b00-6d34-11eb-952f-e6603aa12d0f.png
Doing `.contains("Products → Category")` will not prevent or check for that.

Reply:
It's not **any** `contains`. A very specific element (Notebook cell - which is the green "pill" rounded rectangle) needs to contain this exact string. So this will fail if no notebook cell contains "Products → Category".

Reply:
Cypress should consider using `is()` instead, since it's quite misleading function name.